raiden_contracts/data/source/services/MonitoringService.sol

```diff
@@ -248,27 +248,27 @@ contract MonitoringService is Utils {
         require(service_registry.hasValidRegistration(monitoring_service_address), "service not registered");
 
         TokenNetwork.ChannelState channel_state;
-        uint256 settle_block_number;
-        (settle_block_number, channel_state) = token_network.getChannelInfo(
+        uint256 settle_window;
```

Comment:
Reminder for the naming scheme application (new file). Sorry to be very verbose with these comments. 🙈

Reply:
To be fixed in https://github.com/raiden-network/light-client/issues/3015